core/lib/env_config/src/da_client.rs

```diff
@@ -275,20 +264,17 @@ mod tests {
     fn from_env_eigen_client() {
         let mut lock = MUTEX.lock();
         let config = r#"
-            DA_CLIENT="Eigen"
-            DA_EIGEN_CLIENT_TYPE="Disperser"
-            DA_DISPERSER_RPC="http://localhost:8080"
-            DA_SETTLEMENT_LAYER_CONFIRMATION_DEPTH=0
-            DA_EIGENDA_ETH_RPC="http://localhost:8545"
-            DA_EIGENDA_SVC_MANAGER_ADDRESS="0x123"
-            DA_STATUS_QUERY_TIMEOUT=2
-            DA_STATUS_QUERY_INTERVAL=3
-            DA_WAIT_FOR_FINALIZATION=true
-            DA_AUTHENTICATED=false
-            DA_VERIFY_CERT=false
-            DA_POINTS_SOURCE="Path"
-            DA_POINTS_PATH="resources"
-            DA_CHAIN_ID=1
+            EIGENDA_CLIENT="Eigen"
+            EIGENDA_EIGEN_CLIENT_TYPE="Disperser"
```

Comment:
This test is failling, some values need to be changed:
```suggestion
            DA_CLIENT="Eigen"
```
If I'm not mistaken, `EIGENDA_EIGEN_CLIENT_TYPE` is no longer used, it can be removed.